src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ComboBoxTests/UnoSamples_Tests.ComboBoxTests.cs

```diff
@@ -98,5 +98,26 @@ public void ComboBoxTests_VisibleBounds()
 			// compensates for a possible change of origins with android popups.
 			Assert.AreEqual(popupLocationDifference - sampleControlResultExtended.Rect.Y, popupLocationDifferenceExtended);
 		}
+
+
+		[Test]
+		[AutoRetry]
+		public void ComboBoxTests_Stretch()
+		{
+			Run("UITests.Windows_UI_Xaml_Controls.ComboBox.ComboBox_Stretch");
+
+			var combo01 = _app.Marked("combo01");
+			var sampleControl = _app.Marked("sampleControl");
+			var changeExtended = _app.Marked("changeExtended");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'changeExtended'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5121521)